app/concepts/todo/create.rb

```diff
@@ -0,0 +1,17 @@
+class Todo::Create < Trailblazer::Operation
+	
+	contract do 
+		property :title
+		property :description
+
+		validates :title, presence: true
+		validates :description, presence: true
+	end
+
+	def process(params)
+		@model = params[:current_user].todos.new if params[:current_user]
+		validate(params.except(:current_user), @model) do |todo|
+			todo.save
+		end
+	end
```

Comment:
I'd write this as
```
@model = params[:current_user].todos.new
validate(params[:todo], @model) do |todo_replica|
    todo_replica.save # syncs the replica (i.e. form values) to the actual @model and saves it on the db
end
```
Notice above, we only want to validate params[:todo], not everything in params.

We're trying to avoid instance variables from being littered. So we could refactor this more using Trailblazer shortcuts like so.
```
class Todo::Create < Trailblazer:Operation
    include Model # if you're using trailblazer version < 1.0, it's include CRUD.

    model Todo, :create     # we're telling trailblazer our model is an instance of Todo, :create means don't load params[:id] from the db.
   # if we're coding Todo::Update, we'll use
   # model Todo, :update
   # which will make trailblazer do Todo.find(params[:id])

    contract do
       ...
    end

    def process(params)
        validate(params[:todo]) do |replica| # this line became simpler because of include Model on line #2
            replica.save
        end
    end
```